### ISSUE TITLE
Water sensor _TZ3000_ww9i3e0y

### ISSUE THREAD

**Issue by jleeca** [dresden-elektronik/deconz-rest-plugin#6266]
## Device

- Product name: Water sensor (https://www.aliexpress.com/item/1005003446278673.html?spm=a2g0o.order_list.0.0.15101802pc6qWy)
- Manufacturer: _TZ3000_ww9i3e0y
- Model identifier: TS207
- Device type 
  - Sensor

## Screenshots

![image](https://user-images.githubusercontent.com/20133784/184516952-096b64c9-58d1-400a-a69c-2a84b4f81c6e.png)
![image](https://user-images.githubusercontent.com/20133784/184516978-deac5ce8-494a-4b9f-9451-67a3feaef57c.png)

### Basic
![image](https://user-images.githubusercontent.com/20133784/184516959-35861a5e-46a6-4128-b6b4-c294a7a1ebc4.png)

### Identify
![image](https://user-images.githubusercontent.com/20133784/184516988-cab25e7e-e2ea-463f-bf09-3724370432f0.png)

### Power Configuration
![image](https://user-images.githubusercontent.com/20133784/184517006-a06559ce-910a-4b1d-8d4c-9d9b38f68d88.png)

### IAS Zone
![image](https://user-images.githubusercontent.com/20133784/184517032-68f7b195-2c98-46ab-ba98-a99650965334.png)

**Comment by jleeca** [comment]
FYI, I just tried similar _TZ3000_d16y6col_water_sensor.json DDF file (after replacing the manufacturename to "_TZ3000_ww9i3e0y") but it's not working so it looks like there's more attributes which need to be changed?

**Comment by Smanar** [comment]
There is perhaps a typo in the model id ?

`Model identifier: TS207`

You need to have the "Gold" Status in the DDF

`  "status": "Bronze",`

**Comment by jleeca** [comment]
Changing the Status to "Gold" in the DDF fixed the issue. The model number TS0207 is correct from the original _TZ3000_d16y6col_water_sensor.json DDF file so not needed to modify it.

**Comment by Smanar** [comment]
But you can make a pr for the DDf work for both devices


```
  "manufacturername": ["_TZ3000_d16y6col", "_TZ3000_ww9i3e0y"],
  "modelid": ["TS0207", "TS0207"],
```

I can make the change if you want ?

**Comment by jleeca** [comment]
Yes, I still needed to modify manufacturernams for those two devices in DDF files. I only wanted to clarify that the model number TS0207 is correct. Anyway, please make a pr so others can use those two types of  devices.

**Comment by Smanar** [comment]
Done https://github.com/dresden-elektronik/deconz-rest-plugin/pull/6244

**Comment by maykey22** [comment]
Hi, are there any plans to merge that PR soon?

**Comment by Smanar** [comment]
>on 6 Aug ...

I will go to ask to manup

**Comment by maykey22** [comment]
Any updates? I have this device and would like to use it 😀

**Comment by Smanar** [comment]
No, I m  asking a "roadmap" ATM.
A massive PR merge was in project but there is some problem on recent deconz version, and I think the first objective is correct the blocking issues first.

**Comment by maykey22** [comment]
I got it, IMO you should consider creating smaller PRs. Smaller pieces of code are easier to integrate.

**Comment by Smanar** [comment]
Right, but the problem is lot of device can need the same file, or depend each others.

**Comment by maykey22** [comment]
What is the strategy to deal with this? We try to discover all dependant devices? The consecutive commits to support new devices is the stabilization phase?

**Comment by Smanar** [comment]
manup have asked same ^^. We will have conflict for sure.
For me better to merge small PR from "little" user, because they come rarely. And I can solve confflict on my PR after.
